src/components/profile/Modal.jsx

```diff
@@ -74,17 +78,20 @@ const Modal = ({ picture, setModal }) => {
             <Input
               name="First Name"
               background="bg-code-darkgray"
+              value={info.first}
```

Comment:
these should be placeholders as well and should be hard coded inside since the modal is only being used in the profile section

Reply:
![Uploading image.png…]()

Reply:
<img width="621" alt="image" src="https://github.com/acm-ucr/undecided/assets/122486013/e49f18f2-f341-41d7-a1ba-590fed574173">